MANUAL.md

```diff
@@ -83,67 +83,67 @@ npm install @protobuf-ts/plugin
 yarn add @protobuf-ts/plugin
 ```
 
-This will install the plugin as a dependency in your package.  
-The protocol buffer compiler `protoc` is automatically installed ([explanation](./packages/protoc/README.md)). 
+This will install the plugin as a dependency in your package.
+The protocol buffer compiler `protoc` is automatically installed ([explanation](./packages/protoc/README.md)).
 
 Usage:
 ```shell script
 npx protoc \
   --ts_out src/generated/ \
   --ts_opt long_type_string \
   --proto_path protos \
-  protos/my.proto 
+  protos/my.proto
 ```
 
 
 > **Note:** The generated code requires a runtime package. Install it with:
 > ```shell script
 > # with npm:
 > npm install @protobuf-ts/runtime
-> 
+>
 > # with yarn:
 > yarn add @protobuf-ts/runtime
 > ```
 
 
-> **Note:** By default, the plugin only generates code for the .proto 
-> files you pass as arguments to protoc. If you want to generate code 
+> **Note:** By default, the plugin only generates code for the .proto
+> files you pass as arguments to protoc. If you want to generate code
 > for imported files as well, use the plugin option "generate_dependencies".
-> 
+>
 > ```shell script
 > --ts_opt generate_dependencies
 > ```
 
 
 Available plugin options:
 
-- "long_type_string"  
+- "long_type_string"
```

Comment:
Can you revert the whitespace change in this file? It breaks the formatting of the plugin option list, and maybe other places as well.